src/chat/chat_tests.rs

```diff
@@ -4050,3 +4050,35 @@ async fn test_send_delete_request_no_encryption() -> Result<()> {
     assert_eq!(alice_chat.id.get_msg_cnt(alice).await?, 1);
     Ok(())
 }
+
+/// Tests that in multi-device setup
+/// second device learns the key of a contact
+/// via Autocrypt-Gossip in 1:1 chats.
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn test_oneone_gossip() -> Result<()> {
+    let mut tcm = TestContextManager::new();
+    let alice = &tcm.alice().await;
+    let alice2 = &tcm.alice().await;
+    let bob = &tcm.bob().await;
+
+    tcm.section("Alice imports Bob's vCard and sends a message from the first device");
+    let alice_chat = alice.create_chat(bob).await;
+    let sent_msg = alice.send_text(alice_chat.id, "Hello Bob!").await;
+
+    tcm.section("Alice receives a copy on second device");
+    let rcvd_msg = alice2.recv_msg(&sent_msg).await;
+    assert_eq!(rcvd_msg.get_showpadlock(), true);
```

Comment:
Maybe also check that the contact isn't verified? Maybe this is covered by other tests though